ci-operator/step-registry/ovn/sdn-migration-rollback/ovn-sdn-migration-rollback-commands.sh

```diff
@@ -77,7 +77,7 @@ oc wait mcp --all --for='condition=UPDATING=True' --timeout=300s
 # shellcheck disable=SC2154
```

Comment:
Is this disable still required?
https://www.shellcheck.net/wiki/SC2154

Reply:
nope. good catch. updated.